LastNameI-[RnD-MT]Report.tex

```diff
@@ -61,21 +63,21 @@
 
 \pagestyle{mainmatter}
 
-\subfile{chapters/ch01_introduction}
-\subfile{chapters/ch02_stateoftheart}
-\subfile{chapters/ch03_methodology}
-\subfile{chapters/ch04_solution}
-\subfile{chapters/ch05_evaluation}
-\subfile{chapters/ch06_results}
-\subfile{chapters/ch07_conclusion}
+\include{chapters/ch01_introduction}
```

Comment:
We decided to use subfile, since this allows to compile individual chapters. If you set the main file in TexMaker, is the navigation still a problem?

Reply:
Yes, even if main the file is set as the master file, it doesn't load all the chapters in the sidebar. 
I was not aware of individual file compilation feature of subfile. I guess in that case it makes sense to use subfile.

Reply:
I'm for keeping `subfile` since, as @argenos mentioned, it allows compiling the chapters individually. If `include` is better for TexMaker, it's better to do those changes locally (not everyone is using TexMaker after all).